docker_postgres_init.sql

```diff
@@ -0,0 +1,10 @@
+CREATE USER docker WITH PASSWORD 'password' CREATEDB;
```

Comment:
Just in case, I think Postgres creates a user/pass `postgres` `postgres` by default

Reply:
Correct! Postgres creates a user/pass as default to connect, but i think is a good practice creates a new user as owner of databases.
If you look, I did not modify the user/pass to connect with postgres service